physionet-django/project/models.py

```diff
@@ -1533,28 +1533,179 @@ def get_storage_info(self, force_calculate=True):
             compressed_used=compressed)
 
     def citation_text(self):
+        """
+        Returns the citation of the project in the following styles.
+
+        Parameters
+        ----------
+        styles (list):
+            MLA : MLA (8th edition)
+            APA (default) : APA (7th edition)
+            Chicago : Chicago (17th edition)
+            Harvard : Harvard
+            Vancouver : Vancouver
+
+        Returns
+        -------
+        citation_dict (dictionary) : dictionary of each citation style and
+            their respective citation for the requested project
+        """
+        # For if there are no authors or valuable metadata
         if self.is_legacy:
-            return ''
+            return {'APA':''}
```

Comment:
perhaps clearer to return `{}` or `None` here? The code seems clear, so no need for the comment.

Reply:
This is to resolve the single function refactor while still handling legacy projects. If I return `{}` then code will have to be changed in `views` since right now it is calling `citation_text.APA` as default but if I return `{}` instead then APA wouldn't be defined.

Reply:
I figured calling `citation_text.MLA` or whatever the style is would be easier than editing `views` and adding parameters... That reminds me, I think I need to re-edit the docstrings now.